packages/canary-web-components/src/components/ic-data-table/ic-data-table.tsx

```diff
@@ -1505,7 +1505,12 @@ export class DataTable {
           ({ key }, index) =>
             isSlotUsed(this.el, `${key}-${rowIndex}`) && { key, index }
         )
-        .filter((col) => !!col);
+        .filter(
+          (col) =>
+            !!col &&
+            // skip the column if its already in the row
+            !Object.prototype.hasOwnProperty.call(row, col.key)
```

Comment:
I'm not fan of prototype calls vs methods that already exists on the Object, but it's just a nitpick

Reply:
eslint disagrees: https://eslint.org/docs/latest/rules/no-prototype-builtins#rule-details

Reply:
Okay 👍🏾